lib/npg_pipeline/function/merge_recompress.pm

```diff
@@ -18,12 +16,6 @@ Readonly::Scalar my $FUNCTION_NAME => 'merge_recompress';
 Readonly::Scalar my $BCFTOOLS_TOOL_NAME => 'concat';
 Readonly::Scalar my $BCFTOOLS_INDEX_NAME => 'tabix';
 
```

Comment:
Memory was set here to 2G, which is currently the default, therefore, the memory requirement was not added to the resource definition for the function. Unlike other places in the pipeline, 2G might be required here, so I'd rather we have this explicitly set in the function resources